draft-ietf-mls-protocol.md

```diff
@@ -591,6 +591,22 @@ Group Context:
 Signature Key:
 : A signing key pair used to authenticate the sender of a message.
 
+Proposal:
+: A message that proposes a change to the group, e.g., adding or removing a
+member.
+
+Commit:
+: A message that implements the changes to the group proposed in a set of
+Proposals.
+
+PublicMessage:
+: An MLS protocol message that is signed by its sender and authenticated as
+coming from a member of the group in a particular epoch, but not encrypted.
+
+PrivateMessage:
+: An MLS protocol message that is both signed by its sender and encrypted so
```

Comment:
Suggested edit
"by its sender, authenticated as coming from a member of the group in a particular epoch, and is confidential...". I.e. the definition of PrivateMessage mentions confidentiality but leaves off the authenticity mentioned under PublicMessage (albeit implemented as authenticated encryption).